include/zephyr/drivers/usb/udc.h

```diff
@@ -36,6 +37,8 @@ enum udc_mps0 {
 struct udc_device_caps {
 	/** USB high speed capable controller */
 	uint32_t hs : 1;
+	/** USB high speed capable controller */
```

Comment:
Copy and paste typo here. Should be "super" rather than "high"

Reply:
Good catch, thank you!